<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMessagesDialog</class>
 <widget class="QDialog" name="SendMessagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send Messages</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="verticalFrame">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameAddressFrom">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="addressFromLabel">
        <property name="text">
         <string>&amp;Address From:</string>
        </property>
        <property name="buddy">
         <cstring>addressFrom</cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QValidatedLineEdit" name="addressFrom"/>
        </item>
        <item>
         <widget class="QToolButton" name="addressBookButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Choose address from address book&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../bitcointalkcoin.qrc">
            <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pasteButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Paste address from clipboard&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../bitcointalkcoin.qrc">
            <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+P</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>830</width>
        <height>243</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="entries">
         <property name="spacing">
          <number>6</number>
         </property>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="toolTip">
        <string>Send to multiple recipients at once</string>
       </property>
       <property name="text">
        <string>Add &amp;Recipient</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcointalkcoin.qrc">
         <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Remove all transaction fields</string>
       </property>
       <property name="text">
        <string>Clear &amp;All</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcointalkcoin.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
       <property name="autoRepeatDelay">
        <number>300</number>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Confirm the send action</string>
       </property>
       <property name="text">
        <string>S&amp;end</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcointalkcoin.qrc">
         <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcointalkcoin.qrc"/>
 </resources>
 <connections/>
</ui>
